pyproject.toml

```diff
@@ -12,7 +12,10 @@ repository = "https://github.com/rheinwerk-verlag/postgresql-anonymizer"
 
 [tool.poetry.dependencies]
 python = "~2.7 || ^3.5"
-Faker = "^3.0.0"
+Faker = [
+    { version = "^3.0.0", python = "~2.7"},
+    { version = "^4.8.0", python = "^3.5"}
```

Comment:
As far as I can see Faker [introduced](https://github.com/joke2k/faker/blob/master/CHANGELOG.md#490---2020-10-08) the `unique` feature in 4.9. Maybe you can pin the requirement to at least `^4.9.0`?